### PR TITLE
Fix: Use `PHPUnit\TestFixture` as namespace for test fixtures

### PR DESCRIPTION
**Pull request by localheinz**
This pull request

- [x] uses `PHPUnit\TestFixture` as namespace for test fixtures

Follows https://github.com/sebastianbergmann/phpunit/pull/4907#issuecomment-1061522656.

### CODE REVIEW

File: tests/end-to-end/regression/GitHub/3396/issue-3396-test.phpt [hunk 1]
```diff
@@ -19,34 +19,34 @@ PHPUnit\TextUI\Command::main();
...
+Test 'PHPUnit\TestFixture\DataproviderExecutionOrderTest::testAddNumbersWithADataprovider with data set "1+1=3" (1, 1, 3)' ended
+Test 'PHPUnit\TestFixture\DataproviderExecutionOrderTest::testTestInTheMiddleThatAlwaysWorks' started
+Test 'PHPUnit\TestFixture\DataproviderExecutionOrderTest::testTestInTheMiddleThatAlwaysWorks' ended
+Test 'PHPUnit\TestFixture\DataproviderExecutionOrderTest::testAddMoreNumbersWithADataprovider with data set "1+2=3" (1, 2, 3)' started
+Test 'PHPUnit\TestFixture\DataproviderExecutionOrderTest::testAddMoreNumbersWithADataprovider with data set "1+2=3" (1, 2, 3)' ended
+Test 'PHPUnit\TestFixture\DataproviderExecutionOrderTest::testAddMoreNumbersWithADataprovider with data set "2+1=3" (2, 1, 3)' started
+Test 'PHPUnit\TestFixture\DataproviderExecutionOrderTest::testAddMoreNumbersWithADataprovider with data set "2+1=3" (2, 1, 3)' ended
+Test 'PHPUnit\TestFixture\DataproviderExecutionOrderTest::testAddMoreNumbersWithADataprovider with data set "1+1=3" (1, 1, 3)' started
+Test 'PHPUnit\TestFixture\DataproviderExecutionOrderTest::testAddMoreNumbersWithADataprovider with data set "1+1=3" (1, 1, 3)' ended
```
Comment (localheinz) [inline tests/end-to-end/regression/GitHub/3396/issue-3396-test.phpt:37]: @epdenouden  @sebastianbergmann 

I have no idea why the order would be different when `DataproviderExecutionOrderTest` is moved into the `PHPUnit\TestFixture` namespace. Do you?

Comment (sebastianbergmann) [reply tests/end-to-end/regression/GitHub/3396/issue-3396-test.phpt:37]: No idea. AFAIK, the test no longer exists in `master` so I would not worry about it too much.

**Comment by sebastianbergmann** [comment]
This cannot be merged without conflicts to `master`. Please send a PR with these changes for `master`. Thanks!

**Comment by localheinz** [comment]
Thank you, @sebastianbergmann - will follow up!

**Pull request merged**